orc8r/lib/go/protos/magma_id.go

```diff
@@ -0,0 +1,69 @@
+/*
```

Comment:
I'd prefer the location be something more like `orc8r/lib/go/grpc/` -- this is grpc-related rather than protobuf/model-related